lang/talon/talon-context.talon

```diff
@@ -0,0 +1,21 @@
+tag: user.talonlist
+tag: user.talon
+tag: user.talon_python
+-
+#context requirements
+win require: insert("os: windows\n")
+mac require: insert("os: mac\n")
+linux require: insert("os: linux\n")
+title require: insert("win.title: ")
+application [require] [{user.talon_apps}]:
+    app = talon_apps or ""
+    insert("app: {app}")
```

Comment:
Consider adding `\n` here for consistency — I know you're just moving things around…

Reply:
I believe we'd want to leave the cursor at the correct position when talon_apps is specified. Updated accordingly